Magento_Cms/web/css/source/pages/_404.less

```diff
@@ -1,7 +1,14 @@
+@import 'https://fonts.googleapis.com/css?family=Dosis:300,400,700,800';
```

Comment:
better to do that in XML via `link`. and be sure that the domain isn't blocked by CORS